acorn-loose/src/state.js

```diff
@@ -3,6 +3,8 @@ import {dummyValue} from "./parseutil.js"
 
 function noop() {}
 
+const pluginList = []
```

Comment:
It looks like `pluginList` is going to accumulate all plugins ever loaded during this session, rather than the plugins loaded by a given specific `Parser` class.

Reply:
Fix.